plugins/modules/ovirt_host_network.py

```diff
@@ -281,8 +275,12 @@ def get_type_name(mode_number):
             value=str(mode_number)
         )
     )
+    opts_dict = {}
+    if mode == 4:
+        opts_dict = dict(xmit_hash_policy='2')
+    elif mode < 4:
```

Comment:
Why `elif mode < 4:` or why not `else:`?

Reply:
I found out that in API the 5. and 6. mode does not have miimon.